integration_tests/test_suites/dagster-azure-live-tests/integration_tests/dagster-yamls/access-key-credential.yaml

```diff
@@ -11,4 +11,5 @@ compute_logs:
     prefix: 
       env: TEST_AZURE_LOG_PREFIX
     local_dir: "/tmp/cool"
-    upload_interval: 30    
\ No newline at end of file
+    upload_interval: 30    
+    show_url_only: true
```

Comment:
It seems to be the default. Should you set the others (ex: `default-capture-behavior.yaml` to false? 

Makes me wonder if we're not capturing content we shouldn't. Perhaps worth its own test.

Reply:
`default-capture-behavior` is intended to test the behavior where `show_url_only` is explicitly not set - but I agree for every other one we should probably be setting this value to be explicit.

the default value is show_url_only=False